reports/report.md

```diff
@@ -80,8 +80,8 @@ This report was generated by [Aderyn](https://github.com/Cyfrin/aderyn), a stati
 
```

Comment:
Hmm, I don't see the new detector in this report, despite `StateVariableEvents` contract being present in the repo that aderyn is run on. 🤔

Reply:
I've fixed the other issues but I cannot solve this one, can you guys work out why the detector wouldn't be run? I've followed the instructions on the Detectors Quickstart page to add to `aderyn_core/src/detect/low/mod.rs` and `aderyn_core/src/detect/detector.rs`

Reply:
That's because this detect function has returned Error for all contexts.